Traceback (most recent call last):
  File "GAT.py", line 34, in <module>
    classification_data = classification.prepare_data_for_GAT(args)
  File "../functions/classification.py", line 118, in prepare_data_for_GAT
    X_train = [np.concatenate(d, axis=1) for d in X_train]  # stack: n_epochs, n_channels, n_times
  File "../functions/classification.py", line 118, in <listcomp>
    X_train = [np.concatenate(d, axis=1) for d in X_train]  # stack: n_epochs, n_channels, n_times
  File "<__array_function__ internals>", line 6, in concatenate
ValueError: need at least one array to concatenate
/home/yl254115/miniconda3/lib/python3.7/site-packages/h5py/_hl/dataset.py:313: H5pyDeprecationWarning: dataset.value has been deprecated. Use dataset[()] instead.
  "Use dataset[()] instead.", H5pyDeprecationWarning)
Traceback (most recent call last):
  File "/neurospin/unicog/protocols/intracranial/Syntax_with_Fried/Code/Main/plotting/plot_comparison.py", line 130, in <module>
    channels_with_spikes = data_manip.get_channels_with_spikes_from_combinato_sorted_h5(settings, ['pos']) # TODO: fox 'neg' case
  File "../functions/data_manip.py", line 177, in get_channels_with_spikes_from_combinato_sorted_h5
    f_all_spikes = h5py.File(filename, 'r')
  File "/home/yl254115/miniconda3/lib/python3.7/site-packages/h5py/_hl/files.py", line 394, in __init__
    swmr=swmr)
  File "/home/yl254115/miniconda3/lib/python3.7/site-packages/h5py/_hl/files.py", line 170, in make_fid
    fid = h5f.open(name, flags, fapl=fapl)
  File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "h5py/h5f.pyx", line 85, in h5py.h5f.open
OSError: Unable to open file (unable to lock file, errno = 5, error message = 'Input/output error')
